modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -40,77 +52,32 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
-    <component>:
+    <component>: <1>
       nodeSelector:
-        <node_key>: <node_value>
-        <node_key>: <node_value>
+        <node-label-1> <2>
+        <node-label-2> <3>
         <...>
 ----
-+
-Substitute `<component>` accordingly and substitute `<node_key>: <node_value>` with the map of key-value pairs that specifies a group of destination nodes. Often, only a single key-value pair is used.
-+
-The component can only run on nodes that have each of the specified key-value pairs as labels. The nodes can have additional labels as well.
-+
-[IMPORTANT]
-====
-Many of the monitoring components are deployed by using multiple pods across different nodes in the cluster to maintain high availability. When moving monitoring components to labeled nodes, ensure that enough matching nodes are available to maintain resilience for the component. If only one label is specified, ensure that enough nodes contain that label to distribute all of the pods for the component across separate nodes. Alternatively, you can specify multiple labels each relating to individual nodes.
-====
+<1> Substitute `<component>` with the appropriate monitoring stack component name.
+<2> Substitute `<node-label-1>` with the label you added to the node.
+<3> Optional: Specify additional labels.
+If you specify additional labels, the pods for the component are only scheduled on the nodes that contain all of the specified labels.
 +
 [NOTE]
 ====
 If monitoring components remain in a `Pending` state after configuring the `nodeSelector` constraint, check the pod logs for errors relating to taints and tolerations.
```

Comment:
```suggestion
If monitoring components remain in a `Pending` state after configuring the `nodeSelector` constraint, check the pod events for errors relating to taints and tolerations.
```
if the pod is pending, sometimes we can not check the error from logs, example
```
# oc get pod
NAME       READY   STATUS    RESTARTS   AGE
cpu-demo   0/1     Pending   0          74s

# oc logs cpu-demo
no results

# oc get event | grep cpu-demo
64s         Warning   FailedScheduling                             pod/cpu-demo                                           0/5 nodes are available: 2 node(s) didn't match Pod's node affinity/selector, 3 node(s) had untolerated taint {node-role.kubernetes.io/master: }. preemption: 0/5 nodes are available: 5 Preemption is not helpful for scheduling..
```

Reply:
Changed per your suggestion.